azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -214,12 +214,27 @@ def check_submission_warnings(self, problem: Problem):
                         and resubmitting with a lower timeout."
                 )
 
-    def check_set_schedule(self, schedule: RangeSchedule, schedule_name: str):
+    class ScheduleEvolution(Enum):
+        INCREASING = 1
+        DECREASING = 2
+
+    def check_set_schedule(
+        self, schedule: RangeSchedule, schedule_name: str,
```

Comment:
Please start new variables on a newline (as in the suggestion below) for all other methods as well, thanks!
```suggestion
        self,
        schedule: RangeSchedule,
        schedule_name: str,
```